Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -471,6 +533,39 @@ public static LookupElementBuilder createLookupElement(@NotNull final Project pr
     return lookup;
   }
 
+
+  @NotNull
+  private static CompletionSuggestion createCompletionSuggestionFromAvailableSuggestion(@NotNull final AvailableSuggestion suggestion,
+                                                                                        @NotNull final String kind,
+                                                                                        @NotNull final AvailableSuggestionSet suggestionSet,
+                                                                                        final int relevance) {
+    Element element = suggestion.getElement();
+    return new CompletionSuggestion(
+      kind,
```

Comment:
I removed `kind` argument and passed `"UNKNOWN"` instead. We don't have `CompletionSuggestionKind`, we only have `ElementKind`.